### PR TITLE
Update unittest in gh actions

### PR DESCRIPTION
**Pull request by RaviHari**
Fixes issue #81 

## Proposed Changes
  - Remove lint check for now as it was not part of earlier ci.

### CODE REVIEW

File: .github/workflows/unit-test.yml [hunk 1]
```diff
@@ -20,20 +20,13 @@ jobs:
...
-        uses: golangci/golangci-lint-action@v2
-        with:
-          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.32
-          args: --timeout 2m
-
       - name: Get kubebuilder
         env:
-          version: 3.1.0 # latest stable version
+          version: 2.3.1 # latest stable version
```
Comment (shaoxt) [inline .github/workflows/unit-test.yml:25]: Revert this change ?

**Review by shaoxt — APPROVED** [review]
LGTM

File: .github/workflows/unit-test.yml (hunk 1 above)
Comment (RaviHari) [reply .github/workflows/unit-test.yml:25]: 3.1.0 version is not working. will have to check and fix in next pr.

Comment (RaviHari) [reply .github/workflows/unit-test.yml:25]: 3.1.0 version is not working. will have to check and fix in next pr.
